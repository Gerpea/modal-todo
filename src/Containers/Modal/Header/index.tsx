import React from 'react';

import style from '../styles.module.css';

export interface Props {
  children: React.ReactNode;
}

function ModalHeader({ children }: Props): React.ReactElement {
  return <h2 className={style.Header}>{children}</h2>;
}

export default ModalHeader;
