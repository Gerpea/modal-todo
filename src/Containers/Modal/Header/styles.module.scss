@import '../../../variables';

.Header {
  padding-bottom: 10px;
  border-bottom: 2px solid $c-gray-2;
  font-weight: bold;

  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: baseline;

  padding-right: 1px;
  width: 100%;

  h3 {
    font-size: 1.114rem;
    line-height: 1.45;
    font-weight: bold;

    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  }
}
