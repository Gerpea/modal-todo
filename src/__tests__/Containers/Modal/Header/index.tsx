import React from 'react';
import renderer from 'react-test-renderer';

import Header from 'Containers/Modal/Header';

describe('ModalHeader test', () => {
  it('Header renders', () => {
    const tree = renderer.create(<Header>Header</Header>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
