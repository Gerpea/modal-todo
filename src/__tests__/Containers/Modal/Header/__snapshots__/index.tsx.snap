// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalHeader test Header renders 1`] = `
<h2
  className="Header"
>
  Header
</h2>
`;
